test/browser-tests/precache/serviceworkers/simple.js

```diff
@@ -0,0 +1,30 @@
+/*
+  Copyright 2014 Google Inc. All Rights Reserved.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+'use strict';
+
+/* eslint-env browser */
+
+importScripts('/sw-toolbox.js');
+
+self.toolbox.options.cache = {
+  name: 'precache-valid'
+};
+
+self.toolbox.precache([
```

Comment:
For the most part these tests are copying the qunit tests from what I could see, would you want me to add that test in for this PR or a seperate PR?